recipes/libcpuid/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "libcpuid" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/anrieff/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 36d62842ef43c749c0ba82237b10ede05b298d79a0e39ef5fd1115ba1ff8e126
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("libcpuid") }}
```

```suggestion
    - {{ pin_subpackage("libcpuid", exact=True) }}
```
Looks like the [ABI is *not* stable with the API version number](https://github.com/anrieff/libcpuid/blob/2e61160983f32ba840b2246d3c3850c44626ab0d/configure.ac), so this pinning should be exact. Maybe revisit once the API reaches v1.0

---

Thanks for spotting this, I'll update the recipe.